guacamole/src/main/frontend/src/app/home/styles/home.css

```diff
@@ -79,4 +80,21 @@ a.home-connection, .empty.balancer a.home-connection-group {
 .header-app-name {
     font-size: 0.85em;
     box-shadow: none;
+}
+
+.recent-connections .connection .x-thumbnail {
+    background-color: red;
+    display: flex;
```

Comment:
Hmm, why `flex`?

Reply:
I'm not entirely sure, but if remove it, or set it to something else, it displays like this (white "X" outside of the red box):

![image](https://github.com/apache/guacamole-client/assets/4706000/5e252f42-adc2-46de-a5e1-457d1796dd74)

Reply:
I think it's because the size only gets inherited due to the `flex` property. I was able to get this to work without using `display: flex` (and using an `::after` element instead of an `img` child), which seems more semantically correct:

```
diff --git a/guacamole/src/main/frontend/src/app/home/styles/home.css b/guacamole/src/main/frontend/src/app/home/styles/home.css
index ef3178952..1a4ed9cfa 100644
--- a/guacamole/src/main/frontend/src/app/home/styles/home.css
+++ b/guacamole/src/main/frontend/src/app/home/styles/home.css
@@ -82,9 +82,19 @@ a.home-connection, .empty.balancer a.home-connection-group {
     box-shadow: none;
 }
 
+.recent-connections .connection .x-thumbnail::after {
+    content: '';
+    display: block;
+    height: 100%;
+    width: 100%;
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-position: center center;
+    background-image: url('images/x.svg');
+}
+
 .recent-connections .connection .x-thumbnail {
     background-color: red;
-    display: flex;
     height: 1em;
     width: 1em;
     position: absolute;
@@ -97,4 +107,4 @@ a.home-connection, .empty.balancer a.home-connection-group {
 
 .recent-connections .connection .x-thumbnail:hover {
     opacity: 1.0;
-}
\ No newline at end of file
+}
diff --git a/guacamole/src/main/frontend/src/app/home/templates/guacRecentConnections.html b/guacamole/src/main/frontend/src/app/home/templates/guacRecentConnections.html
index 97207dbd1..50e07e623 100644
--- a/guacamole/src/main/frontend/src/app/home/templates/guacRecentConnections.html
+++ b/guacamole/src/main/frontend/src/app/home/templates/guacRecentConnections.html
@@ -20,7 +20,6 @@
         </a>
         <!-- Remove thumbnail -->
         <div class="x-thumbnail" ng-click="removeRecentConnection(recentConnection)">
-            <img alt="X" src="images/x.svg"/>
         </div>
     </div>
```

Reply:
Cool - reworked it with your suggest CSS code.